docker-database/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.8'
+services:
+  db:
+    container_name: database
+    build: DB_dockerDir/
+    # restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - '5432:5432'
+    volumes: 
+      - db:/var/lib/postgresql/data
```

Comment:
I understand you want an ephemeral setup. If that is correct then you might want to avoid defining a named volume (and let docker compose manage all for you) or use a tmpfs.

Reply:
sounds wise. Need some time to investigate the best way to do this

Reply:
done